scipy/linalg/_solvers.py

```diff
@@ -150,7 +150,7 @@ def _solve_discrete_lyapunov_bilinear(a, q):
 
 def solve_discrete_lyapunov(a, q, method=None):
     """
-    Solves the discrete Lyapunov equation :math:`A'XA-X=-Q`.
+    Solves the discrete Lyapunov equation :math:`AXA^H - X = -Q`.
```

Comment:
Wouldn't it better to write the equation as `AXA^H - X  + Q = 0`, then the description of `q` can be change. 

I found defining something as "negative of the right-hand side" to be quite weird. Some meaningful definition perhaps can be give to `Q` (although I don't remember a lot about Lyapunov equation now).